REvernus/Views/MainWindowView.xaml

```diff
@@ -49,7 +49,18 @@
             <Grid>
                 <TabControl x:Name="MainTabControl">
                     <TabItem Header="Market Orders">
-                        <views:MarketOrdersViewerView/>
+                        <TabControl>
+                            <TabItem Header="Buy/Sell">
+                                <views:MarketOrdersViewerView/>
+                            </TabItem>
+                            <TabItem Header="Buy">
```

Comment:
What is the purpose of these TabItems? They seem like they should be in MarketOrderViewerView.xaml, not MainWindowView.xaml

Reply:
Removed